### PR TITLE
Add using example

### PR DESCRIPTION
**Pull request by MetalDent**
Fixes dotnet/docs[#14869](https://github.com/dotnet/docs/issues/14869).

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR/RegularExpressions.Examples.ChangeDateFormats/cs/Example_ChangeDateFormats1.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ static string MDYToDMY(string input)
    {
       try {
          return Regex.Replace(input, 
-               "\\b(?<month>\\d{1,2})/(?<day>\\d{1,2})/(?<year>\\d{2,4})\\b",
+                @"\b(?<month>\d{1,2})/(?<day>\d{1,2})/(?<year>\d{2,4})\b"
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_CLR/RegularExpressions.Examples.ChangeDateFormats/cs/Example_ChangeDateFormats1.cs]: The change for the `using` statement looks great. For this PR, we don't want this change included, as it's in another PR.

Can you revert this commit on this branch?

**Comment by BillWagner** [comment]
@MetalDent This has gotten stale.

I like the sample that was added, but it is not referenced in the article. Do you want to update the article to include the new sample in a PR in [dotnet/docs](https://github.com/dotnet/docs)? Otherwise, I will close this.

**Comment by MetalDent** [comment]
> @MetalDent This has gotten stale.
> 
> I like the sample that was added, but it is not referenced in the article. Do you want to update the article to include the new sample in a PR in [dotnet/docs](https://github.com/dotnet/docs)? Otherwise, I will close this.

Okay, I'll update the article

**Comment by BillWagner** [comment]
Closing due to lack of response.

@MetalDent If you have time to revisit this, I'll watch for any updates.

**Pull request closed without merging**